docs/v1/models.md

```diff
@@ -17,9 +17,9 @@
 ### Transaction
 + hash: `ghf6msRhE4jRf5DPib9UHD1msadvmZs9o53V9FQTb11` (string,required)
 + source: `GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ` (string,required) 
-+ fee: `10000` (string,required)
++ fee: `10000` (string,required) - when user sent money to another account, fee have to be charged. 
 + sequence_id: 0 (number)
-+ created: `2018-09-12T09:08:35.157472400Z` 
++ created: `2018-09-12T09:08:35.157472400Z` - transaction created time. 
```

Comment:
I am not sure this is transaction created time or received time or confirmed time.
But it would be better to be set exactly what the `created time` is

Reply:
then I have to be checked source code. I will check it.